elements/bulbs-video/elements/carousel-buttons.test.js

```diff
@@ -0,0 +1,135 @@
+import './carousel-buttons';
+let subject;
+let carousel;
+let container;
+
+function carouselButtonExamples () {
+  describe('createdCallback', () => {
+    it('sets innerHTML', () => {
```

Comment:
We should finish this test and at the very least mark as pending with `xit` so we it's not a presumed green test